BUILD.md

```diff
@@ -38,10 +38,27 @@ which includes compiler support and dependent libraries.
 Compiling
 ---------
 
+We use virtualenv to install required python packages into the build directory.
+If you don't already have it, you can install it with:
+
     pip install virtualenv
 
-We use virtualenv to install required python packages into the build directory,
-be sure to install virtualenv using your system pip (conda pip is not supported)
+Note that you may need to do a system-wide install with `sudo`; this depends on your Python environment and whether your `pip` binary requires sudo permissions.
```

Comment:
Worth noting that you can also use `pip install --user`?

Reply:
Hmm. Probably a good call but I haven't tried it. Is there any downside to `pip install --user` for most users?

Reply:
On Mac, it meant that I had to add ~/Library/Python/2.7/bin to my PATH, but I haven't noticed any real downsides. In my head, it fits into the spirit of virtualenv by allowing me to be as agnostic as possible to whatever the "system" installation is.